.buildkite/longruns_gpu/pipeline.yml

```diff
@@ -37,7 +37,7 @@ steps:
         artifact_paths: "land_longrun_gpu/*png"
         agents:
           slurm_gpus: 1
-          slurm_time: 03:00:00
+          slurm_time: 08:00:00
```

Comment:
do we need this change? it looks like the run is still taking ~2.5 hours to run

Reply:
thanks!